components/src/main/res/values/attrs.xml

```diff
@@ -37,136 +55,216 @@
+        <!-- Title to display. -->
         <attr name="title" />
+        <!-- Title content description. -->
         <attr name="titleContentDescription" format="string" />
+        <!-- Subtitle to display. -->
         <attr name="subtitle" />
+        <!-- Subtitle content description. -->
         <attr name="subtitleContentDescription" format="string" />
+        <!-- Icon to show at the start of the expanding row. -->
         <attr name="icon" />
+        <!-- Whether or not a click anywhere toggles expansion. If true then anywhere can be clicked to toggle expansion. If false then the header must be clicked to toggle expansion. -->
         <attr name="clickTogglesExpansion" format="boolean" />
+        <!-- Whether or not the view should begin as expanded -->
         <attr name="expanded" format="boolean" />
     </declare-styleable>
 
     <declare-styleable name="IconButtonView">
+        <!-- Title to display. -->
         <attr name="title" />
+        <!-- Icon to display. -->
         <attr name="icon" />
+        <!-- Accessibility message to read. -->
         <attr name="accessibilityMessage" />
     </declare-styleable>
 
     <declare-styleable name="TitleBodyView">
+        <!-- Title to display. -->
         <attr name="title" />
+        <!-- Body text to display. -->
         <attr name="body" format="string" />
     </declare-styleable>
 
+    <!-- Hello. -->
     <declare-styleable name="MultiColumnRowView">
+        <!-- First piece of text to display. -->
         <attr name="text1" format="string" />
+        <!-- Second piece of text to display. -->
         <attr name="text2" format="string" />
+        <!-- Third piece of text to display. -->
         <attr name="text3" format="string" />
+        <!-- First piece of text content description. -->
         <attr name="text1ContentDescription" format="string" />
+        <!-- Second piece of text content description. -->
         <attr name="text2ContentDescription" format="string" />
+        <!-- Third piece of text content description. -->
         <attr name="text3ContentDescription" format="string" />
+        <!-- Whether or not the first piece of text is selectable. -->
         <attr name="text1IsSelectable" format="boolean" />
+        <!-- Whether or not the second piece of text is selectable. -->
         <attr name="text2IsSelectable" format="boolean" />
+        <!-- Whether or not the third piece of text is selectable. -->
         <attr name="text3IsSelectable" format="boolean" />
+        <!-- Text style for the first piece of text. -->
         <attr name="textStyle" format="reference" />
+        <!-- Text style for the second piece of text. -->
         <attr name="textStyle2" format="reference" />
+        <!-- Text style for the third piece of text. -->
         <attr name="textStyle3" format="reference" />
+        <!-- Whether or not the first piece of text should be rendered as a heading. -->
```

Comment:
I think this attribute sets text 1 as an accessibility heading, doesn't change how it is rendered.

Reply:
Great spot! Fixed in https://github.com/hmrc/android-components/pull/21/commits/a00d8142e0abb6afd22971b20b2a6580983f104f